index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css" rel="stylesheet">
+  <link href="style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400" rel="stylesheet">
+</head>
+<body>
+  <div class="calculator">
+    <div class="calculator-screen">0</div>
+    <div class="calculator-row">
+      <button class="calculator-button calculator-AC">AC</button>
+      <button class="calculator-button calculator-positive-negative">+/-</button>
+      <button class="calculator-button calculator-percentage">%</button>
+      <button class="calculator-button calculator-divide">÷</button>
+    </div>
+    <div class="calculator-row">
+      <button class="calculator-button calculator-seven">7</button>
```

Comment:
Good html organization here.

I would recommend a separate div around all your calculator rows.  When you get to adding the event listener, it will help to have one big div for all the buttons.